casepro/statistics/signals.py

```diff
@@ -38,6 +41,29 @@ def record_new_outgoing(sender, instance, created, **kwargs):
         if instance.partner:
             DailyCount.record_item(day, DailyCount.TYPE_REPLIES, partner)
 
+        if case:
+            # count the very first response on an org level
+            if instance == case.outgoing_messages.earliest('created_on'):
```

Comment:
This is recording the time between a case being opened and the first reply sent by a user. Aren't we actually interested in the time between an incoming message being received and the first reply? That's certainly what I assumed from the header "Average time to reply" on the org dashboard.

If partners are opening their own cases, and immediately sending a reply - then this number will make it look like the partner is very responsive - but the incoming messages may have been sitting there for a long time with no reply.